README.md

```diff
@@ -1,2 +1,23 @@
 # xrviz
 Interactive visualisation interface for Xarrays
+
+## Installation
+
+### Using Conda
+```
+conda install -c conda-forge xrviz
+pip install git+https://github.com/intake/xrviz --no-deps
+```
+
+### Using Pip
+```
+git clone https://github.com/intake/xrviz.git
```

Comment:
these three lines are equivalent to `pip install pip+https://github.com/intake/xrviz.git`. Have you tested it in a fresh environment with the current requirements.txt?

Reply:
Yes I have tested installing via pip in a new environment, used the interface and ran tests(after installing pytest).

Reply:
We may wish to have one of the travis test runs which installs from pip, like Intake does: https://github.com/intake/intake/blob/master/.travis.yml#L28

Reply:
I have tried to figure this out, structure of both travis files look a bit different to me. For this task I need your help. Could this be done in a separate PR?